src/strategies/balance-of-subgraph/index.ts

```diff
@@ -0,0 +1,55 @@
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest } from '../../utils';
+
+const SUBGRAPH_URL = {
+  '1': 'https://api.thegraph.com/subgraphs/name/dinngodev/furucombo-tokenomics-mainnet',
+  '137':
+    'https://api.thegraph.com/subgraphs/name/dinngodev/furucombo-tokenomics-polygon'
+};
+
+export const author = 'weizard';
+export const version = '0.1.0';
+
+export async function strategy(
+  _space,
+  network,
+  _provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const params = {
+    users: {
+      __args: {
+        where: {
+          id_in: addresses.map((address) => address.toLowerCase()),
+          amount_gt: 0
+        }
+      },
+      id: true,
+      amount: true
```

Comment:
Hi @weizard Looks good, but would be great if there is a way to pass optional param names for input and output from options. can check if there is a possibility.?

Reply:
Did you mean that add token address and symbol into the option and filter the address in the subgraph ?

Reply:
I mean names of the params, `amount` and `id_in`

Reply:
I am still not sure I actually understand, is it like that option has a property named `subGraphAddreeKey`, and its value may be `id`.
options will be
```
{
  "subGraphURL": "https://api.thegraph.com/subgraphs/name/dinngodev/furucombo-tokenomics-mainnet",
  "subGraphAddreeKey": "id"
}
```
And line 25 will be like
``` 
[`${options. subGraphAddreeKey ?? 'id'}_in`]: addresses.map((address) => address.toLowerCase()),
```